index.html

```diff
@@ -0,0 +1,138 @@
+              <h2 class="to-try__header">Food</h2>
+              <p class="to-try__paragraph to-try__paragraph__list">Always order a Brand Burger, you will not be disappointed!</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- THERE IS THE BRIDGE OF THE DEVIL - No. 2 - Yes, you can go through it! If... -->
+        <div class="reasons__inner reasons__inner--blue">
+          <h2 class="reasons__header">There's a Devil's Bridge</h2>
+          <p class="reasons__paragraph">— №2 —</p>
+          <p class="reasons__paragraph">Yes, you can go through it! If you dare of course.</p>
+        </div>
+        <div class="reasons__background-img"></div>
+      </section>
+
+      <section class="reasons"><!-- SMALL AREA - No. 3 - ALL SITES are located -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Sall Area</h2>
+          <p class="reasons__paragraph">— №3 —</p>
+          <p class="reasons__paragraph">All the attractions are very close</p>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- BEAUTIFUL ROAD - No. 4 - TO GO TO SEDONU FROM LAS VEGAS completely... -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Beautiful Road</h2>
+          <p class="reasons__paragraph">— №4 —</p>
+          <p class="reasons__paragraph">Go to Sedona Las Vegas it's anything but not boring!</p>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- LITTLE TOURISTS - No. 5 - Most travel to Grand Canyon and crowd... -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Small Tourism</h2>
+          <p class="reasons__paragraph">— №5 —</p>
+          <p class="reasons__paragraph">Most of the turrist travel to the Grand Canyon and it is over crowded. In here is oposite!</p>
+        </div>
+      </section>
+
+      <section class="search"><!-- INTERESTED? Indicate the expected dates of your trip and we are -->
+        <div class="search__hotel">
+          <h2 class="search__header">Intereted?</h1>
+          <p class="search__paragraph">Specify the expected dates and we will show you the best deals for hotels in Sedona</p>
+
+          <button class="btn btn__search">Find your hotel</button>
+        </div>
+      </section>
+
+      <section class="map">
+        <img class="image__map"><!-- map -->
+      </section>
+
+      <footer class="page-footer">
+        <div class="footer-wrapper">
+          <div class="page-footer__tag">
+            <h2 class="page-footer__tag-header">#VISITSEDONA</h2>
+          </div>
+
+          <div class="page-footer__social">
+            <a href="" class="page-footer__img--tweeter"></a>
+            <a href="" class="page-footer__img--facebook"></a>
+            <a href="" class="page-footer__img--youtube"></a>
+          </div>
+
+          <div class="page-footer__developer">
+            <p class="footer__paragraph">Developed by</p>
```

Comment:
don't think it should be a paragraph. Should it be an inline element maybe? Should it be hoverable with the logo?